pages/signup/index.tsx

```diff
@@ -31,13 +31,15 @@ const SignUpPage = ({ t }: PageProps): React.ReactElement => {
     name: {
       lastName: '',
     },
-    gender: Gender.OTHER,
```

Comment:
I have the required changes on my computer. Will push directly to main once this pr is merged, if that's ok.

Reply:
ok, fine for me.